configuration/initialize_docker_environment.sh

```diff
@@ -9,6 +9,8 @@ SIMULATION_MODE_DEFAULT="FALSE"
 SIMULATION_IP_DEFAULT="127.0.0.1"
 SENSOR_JSON_FILE_PATH_DEFAULT="/var/www/plugins/MAP/sensors.json"
 COMPOSE_PROFILES=""
+PLUGIN_INPUT_DEFAULT="All"
```

Comment:
So to properly integrate this with the initialization.sh script you need to provide users with a select to build or pull the V2X Hub docker image. If they select to build it, then you can prompt them to only build certain selected plugins. The way you currently have it configured, It will always attempt to build V2X Hub locally which is incorrect.

If adding this to initialization.sh script is too large of scope right now, create a follow on story for that and remove changes to the `configuration` directory from this PR.